pkg/service/replay/replay.go

```diff
@@ -160,12 +160,16 @@ func (r *Replayer) Start(ctx context.Context) error {
 	}
 	if !r.config.Test.SkipCoverage {
 		if utils.CmdType(r.config.CommandType) == utils.Native {
-			r.config.Command, err = cov.PreProcess()
+			r.config.Command, err = cov.PreProcess(r.config.Test.SkipPreview)
 
 			if err != nil {
 				r.config.Test.SkipCoverage = true
 			}
+
 		}
+	}
+
+	if !r.config.Test.SkipCoverage {
 		err = os.Setenv("CLEAN", "true") // related to javascript coverage calculation
```

Comment:
why is this needed?

Reply:
I had done this change for my previous approach, currently it doesn't have any need. Should I revert to previous syntax?